### ISSUE TITLE
I'd like to have a seperation on suites

### ISSUE THREAD

**Issue by babielph** [gextech/grunt-nightwatch#60]
Hey there,

I found a way how to seperate my tasks with your grunt plugin.
`grunt nightwatch:chrome --group <folder>` 
is helping a lot. But since I'm using Grunt because of their tasks I'd like to use this feature by adding a configuration into the Gruntfile.js

Can you please point out how to do this? Also I'd like to say that it's a bit hard to follow your documentation about "targets". Would be awesome if you would write a full configurated documentation for this feature.

greets Charlie

**Comment by pateketrueke** [comment]
Sure, I'll work on that asap, thanks for your feedback!

**Comment by YoungElPaso** [comment]
I dunno if this is relevant but just to say that I used the 'excluded' config option and dynamically set that in Grunt depending on grunt options that are passed in via the command line.  I think it achieves what is desired in specifying groups.

**Comment by pateketrueke** [comment]
A bit of clarification on targets: https://github.com/gextech/grunt-nightwatch/issues/62#issuecomment-219343822
